cluster_kwargs.yaml

```diff
@@ -66,3 +66,10 @@ test_work_stealing_on_straggling_worker:
 test_repeated_merge_spill:
   n_workers: 20
   worker_vm_types: [m6i.large]
+
+# For tests/workflows/test_from_csv_to_parquet.py
+from_csv_to_parquet_cluster:
+  n_workers: 30
+  worker_vm_types: [t3.medium]  # 2CPU, 4GiB
```

Comment:
Are we not using the default instances on purpose here, is there a reason for using the burstable ones?

Reply:
No reason other than ignorance. :sweat_smile: